### ISSUE TITLE
Error generating and trying to edit module tokens

### ISSUE THREAD

**Issue by TomAbel** [sendsmaily/smaily-prestashop-module#33]
**Bug description**
Customer Sync token & Abandoned Cart token are not automatically generated and cannot be edited.

**Steps for reproducing**

1. Install module
2. Enable debug mode
3. Go to module configuration page

**Expected behavior**
No error

**Actual behavior**
```
Notice on line 191 in file /var/www/html/var/cache/dev/smarty/compile/ae/c3/83/aec383646e18dca4c964335a2a6f4d1b8f9bc531_0.file.smaily_configure.tpl.php
[8] Undefined index: smaily_customer_cron_token

Notice on line 191 in file /var/www/html/var/cache/dev/smarty/compile/ae/c3/83/aec383646e18dca4c964335a2a6f4d1b8f9bc531_0.file.smaily_configure.tpl.php
[8] Trying to get property 'value' of non-object

Notice on line 362 in file /var/www/html/var/cache/dev/smarty/compile/ae/c3/83/aec383646e18dca4c964335a2a6f4d1b8f9bc531_0.file.smaily_configure.tpl.php
[8] Undefined index: smaily_cart_cron_token

Notice on line 362 in file /var/www/html/var/cache/dev/smarty/compile/ae/c3/83/aec383646e18dca4c964335a2a6f4d1b8f9bc531_0.file.smaily_configure.tpl.php
[8] Trying to get property 'value' of non-object
```

**Screenshots**
![image](https://user-images.githubusercontent.com/33309474/97878245-0e8d3f80-1d27-11eb-8771-e78bc613c22b.png)

**Prestashop version**
v1.7.6.38

**Module version**
v1.4.0

**Additional context**
Notice on line x in file may be different in screenshot. I was in the process of editing these files.
Tried a clean install on my laptop, same issue.

**Comment by TomAbel** [comment]
I copied the compiled template file from the container docker to my PC.
The error is on these lines
191: ```
<input type="text" name="SMAILY_CUSTOMER_CRON_TOKEN" id="SMAILY_CUSTOMER_CRON_TOKEN" value="<?php echo $_smarty_tpl->tpl_vars['smaily_customer_cron_token']->value;?>
">
```
404: ```
<input type="text" name="SMAILY_CART_CRON_TOKEN" id="SMAILY_CART_CRON_TOKEN" value="<?php echo $_smarty_tpl->tpl_vars['smaily_cart_cron_token']->value;?>
">
```

**Comment by kaittodesk** [comment]
Close this issue, if it is resolved.

**Comment by TomAbel** [comment]
Closed by #35
